recipes/sentinelhub/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "sentinelhub" %}
+{% set version = "2.5.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7665fb20583130ccc4a30f958ad47e961a7a911e95a687b0b25a4211bc3b028d"
+
+build:
+  number: 0
+  entry_points:
+    - sentinelhub=sentinelhub.commands:main_help
+    - sentinelhub.aws=sentinelhub.commands:aws
+    - sentinelhub.config=sentinelhub.commands:config
+    - sentinelhub.download=sentinelhub.commands:download
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py<32]
+
+requirements:
+  host:
+    - boto3
+    - botocore
+    - click
+    - numpy
+    - pillow
+    - pip
+    - pyproj
+    - python
+    - python-dateutil
+    - requests
+    - shapely
+    - tifffile
+    - utm
+    - wheel
+  run:
+    - boto3
+    - botocore
+    - click
+    - numpy
+    - pillow
+    - pyproj
+    - python
+    - python-dateutil
+    - requests
+    - shapely
+    - tifffile
+    - utm
+    - wheel
+
+test:
+  imports:
+    - sentinelhub
+  commands:
+    - sentinelhub --help
+    - sentinelhub.aws --help
+    - sentinelhub.config --help
+    - sentinelhub.download --help
+
+about:
+  home: "https://github.com/sentinel-hub/sentinelhub-py"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE.md"
+  summary: "Sentinel Hub Utilities: skeleton by Thingus"
+  doc_url: ""
```

Comment:
- doc_url is https://sentinelhub-py.readthedocs.io/en/latest/
- dev_url is https://github.com/sentinel-hub/sentinelhub-py

Reply:
Addressed; thanks!